google/cloud/bigtable/rpc_retry_policy.h

```diff
@@ -24,6 +25,25 @@ namespace google {
 namespace cloud {
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
+/// An adapter to use `grpc::Status` with the `google::cloud::*Policies`.
+struct SafeGrpcRetry {
+  // Sometimes we need to use google::protobuf::rpc::Status, in which case this
+  // is a easier function
+  static inline bool IsTransientFailure(grpc::StatusCode code) {
```

Comment:
Aren't `inline` decorators ignored by compilers these days anyway?

Reply:
They `inline` is ignored with respect to whether the function is inlined or not.  But it is important because without it the function would be in violation of the ODR.  Non-inline functions must be defined in a single translation unit.  Inline functions definitions can appear in multiple translation units.  In other words, if you are going to *define* a non-template function in a header file, you must declare it `inline` or it is a ODR violation (unless the header is used in a single `.cc` file, but I digress).

Reply:
TIL. Neat.